executor/index_lookup_join.go

```diff
@@ -590,6 +588,9 @@ func (iw *innerWorker) fetchInnerResults(ctx context.Context, task *lookUpJoinTa
 	innerResult.GetMemTracker().SetLabel(innerResultLabel)
 	innerResult.GetMemTracker().AttachTo(task.memTracker)
 	for {
+		if len(ctx.Done()) > 0 {
+			return nil
+		}
```

Comment:
We must use select here because the length of `ctx.Done()` may always be 0, see https://play.golang.org/p/bxnZutqxLhF